### ISSUE TITLE
Feature Request: Allow styling of the listItemComponent

### ISSUE THREAD

**Issue by jannnik** [minop1205/react-dnd-treeview#59]
# Feature request

## Summary

Allow styling of the listItemComponent when it is not a dragSource or dropTarget. 

## Purpose / Reason

We want to add a dashed 1px border to the listItemComponent when it is a dropTarget. To prevent the items from jumping around while dragging, we also need to add a transparent 1px border to all listItemComponents. Unfortunately, it is not possible to style the listItemComponent via the `classes` property. 

## If you have an implementation idea.

Allow setting default styles of the listItemComponent via the `classes` property. 

## Reference information

https://github.com/minop1205/react-dnd-treeview#component-styling

![image](https://user-images.githubusercontent.com/7594388/131703514-61fb1da9-9abe-4a8f-af28-eb7b610a4a06.png)

**Comment by minop1205** [comment]
@jannnik Thank you for the feature request.

I have just implemented the feature in the following CodeSandbox, is it working as you intended?

https://codesandbox.io/s/full-features-issue-59-twpze

I made it possible to specify classes.listItem.
I also specified classes.root to have the same effect for dropping to the root.

The following two files are related to this feature.
https://codesandbox.io/s/full-features-issue-59-twpze?file=/src/App.tsx
https://codesandbox.io/s/full-features-issue-59-twpze?file=/src/App.module.css

If this is OK, publish it as v1.5.3.
